Orso.Arpa.Application/Localization/StringLocalizer.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using AutoMapper.Internal;
+using Microsoft.Extensions.Localization;
+
+namespace Orso.Arpa.Application.Localization
+{
+    public class StringLocalizer : IStringLocalizer
+    {
+        private readonly string _resourceKey;
+        private readonly string _location;
+        private readonly LocalizerCache _cache;
+
+        public StringLocalizer(string resourceKey, string location, LocalizerCache cache)
+        {
+            _resourceKey = resourceKey;
+            _location = location;
+            _cache = cache;
+        }
+
+        public IEnumerable<LocalizedString> GetAllStrings(bool includeParentCultures)
+        {
+            IList<LocalizedString> localizedStrings = new List<LocalizedString>();
+
+            CultureInfo culture = CultureInfo.GetCultures(CultureTypes.AllCultures).AsQueryable()
+                .First(q => q.Name.Equals(_location));
```

Comment:
Warum nicht einfach new CultureInfo(_location)?

Reply:
Funktioniert das denn für "en" und "de"?

Reply:
Stimmt. Das ist besser.
Funktioniert solange in der Datenbank "de" bzw. "en" eingetragen ist.